app/views/layouts/_item.html.haml

```diff
@@ -57,9 +57,11 @@
         .col-md-8
           - item_data = @item.send(item)
           - if %w(data name).include?(@view.headers[i].downcase)
-            - if item_data.present?
+            - if item_data.present? && !(item_data.kind_of?(TrueClass) || item_data.kind_of?(FalseClass))
```

Comment:
This seems a lot simpler to just see if `item_data.respond_to?(:scan)` (you don't even need the .present? check if you do that, I think)